src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IBrokeredDebuggerServices.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework;
+using Microsoft.VisualStudio.Debugger.Contracts.HotReload;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Contracts
+{
+    /// <summary>
+    /// Facade interface for getting various service brokers
+    /// </summary>
+    internal interface IBrokeredDebuggerServices
```

Comment:
💡 All of the types defined in this assembly should start with `IVisualDiagnostics` or `VisualDiagnostics` to avoid ambiguity when attempting to navigate to an item by name in Roslyn.sln.